proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -46,6 +47,8 @@ message GenericSignedBeaconBlock {
 
         // Representing a signed, post-Capella fork blinded beacon block.
         SignedBlindedBeaconBlockCapella blinded_capella = 6;
+
+        SignedBeaconBlockDenebAndBlobs Deneb  = 7;
```

Comment:
Not that I care much about names, but I find this one confusing. `SignedBeaconBlockAndBlobsDeneb` may be better to parse but still is also not nice.

Reply:
13b966725fb78420f8d2a70412952d365942a15e